frontend/src/metabase/timelines/questions/components/TimelineCard/TimelineCard.unit.spec.tsx

```diff
@@ -39,28 +39,14 @@ describe("TimelineCard", () => {
 
     expect(props.onToggleTimeline).toHaveBeenCalled();
   });
-
-  it("should make a timeline visible when its event is selected", () => {
-    const props = getProps({
-      timeline: createMockTimeline({
-        name: "Releases",
-        events: [createMockTimelineEvent({ name: "RC" })],
-      }),
-      isVisible: false,
-    });
-
-    render(<TimelineCard {...props} />);
-    userEvent.click(screen.getByText("Releases"));
-    userEvent.click(screen.getByText("RC"));
-
-    expect(props.onToggleTimeline).toHaveBeenCalledWith(props.timeline, true);
-  });
 });
 
 const getProps = (opts?: Partial<TimelineCardProps>): TimelineCardProps => ({
   timeline: createMockTimeline(),
+  visibleEventIds: [],
```

Comment:
This and other addition to `getProps` functions in unit test files in this PR reflect added requirements in types used in the corresponding implementation files.